tests/unit/plugins/module_utils/test_postgres.py

```diff
@@ -31,7 +31,7 @@
     user=dict(default='postgres'),
     password=dict(default='test', no_log=True),
     host=dict(default='test'),
-    port=dict(type='int', default=5432, aliases=['login_port']),
+    port=dict(type='int', aliases=['port'], default=5432),
```

Comment:
Is the variable name supposed to be `port` or `login_port`.

Reply:
@andreasscherbaum this dict is used in the test_get_conn_params_def unit test that tests what the `get_conn_params` function. The function returns what postgres understands, i.e. user, password, port.
So i believe this test code is correct.

Reply:
it looks confusing, yes, was to me too, we should probably rewrite the logic one day